packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.test.tsx

```diff
@@ -1,318 +1,56 @@
-      const expectedMsgId = monthBasedId;
-
-      it('Handles an interval count of 1', () => {
-        const plan_id = 'plan_monthly';
-        const plan = findMockPlan(plan_id);
-        const periodEndDate = getLocalizedDateString(
-          subscription.current_period_end,
-          true
-        );
-        const expectedMsg = `You are billed $5.00 monthly for FPN. Your next payment occurs on ${periodEndDate}.`;
-
-        const props = {
-          ...baseProps,
-          plan: plan,
-        };
-        runTests(props, expectedMsgId, expectedMsg);
-      });
-
-      it('Handles an interval count that is not 1', () => {
-        const plan_id = 'plan_6months';
-        const plan = findMockPlan(plan_id);
-        const periodEndDate = getLocalizedDateString(
-          subscription.current_period_end,
-          true
-        );
-        const expectedMsg = `You are billed $5.00 every 6 months for FPN. Your next payment occurs on ${periodEndDate}.`;
-
-        const props = {
-          ...baseProps,
-          plan: plan,
-        };
-        runTests(props, expectedMsgId, expectedMsg);
-      });
-    });
-
-    describe('When plan has year interval', () => {
-      const expectedMsgId = yearBasedId;
-
-      it('Handles an interval count of 1', () => {
-        const plan_id = 'plan_yearly';
-        const plan = findMockPlan(plan_id);
-        const periodEndDate = getLocalizedDateString(
-          subscription.current_period_end,
-          true
-        );
-        const expectedMsg = `You are billed $5.00 yearly for FPN. Your next payment occurs on ${periodEndDate}.`;
-
-        const props = {
-          ...baseProps,
-          plan: plan,
-        };
-        runTests(props, expectedMsgId, expectedMsg);
-      });
-
-      it('Handles an interval count that is not 1', () => {
-        const plan_id = 'plan_6years';
-        const plan = findMockPlan(plan_id);
-        const periodEndDate = getLocalizedDateString(
-          subscription.current_period_end,
-          true
-        );
-        const expectedMsg = `You are billed $5.00 every 6 years for FPN. Your next payment occurs on ${periodEndDate}.`;
-
-        const props = {
-          ...baseProps,
-          plan: plan,
-        };
-        runTests(props, expectedMsgId, expectedMsg);
-      });
-    });
+import PaymentForm from '../../components/PaymentForm';
+jest.mock('../../components/PaymentForm', () => {
+  return jest.fn().mockImplementation(() => <div></div>);
+});
+import { PaymentUpdateForm, PaymentUpdateFormProps } from './PaymentUpdateForm';
+import { CUSTOMER } from '../../lib/mock-data';
+import defaultState from 'fxa-payments-server/src/store/state';
+
+const { queryByTestId, queryByText, getByTestId } = screen;
+
+const baseProps: PaymentUpdateFormProps = {
+  customer: CUSTOMER,
+  resetUpdatePayment: jest.fn(),
+  updatePayment: jest.fn().mockResolvedValue(null),
+  updatePaymentStatus: defaultState.updatePayment,
+};
+
```

Comment:
As mentioned above (if you are viewing this on GitHub), the tests have been replaced and moved: https://github.com/mozilla/fxa/pull/6144/files#r465808631

I added a couple simple rendering tests below.  Keep in mind that:
 - the payment form has its own tests in `src/components/PaymentForm/index.test.tsx`
 - we are planning to remove all the /v1 code, so expending effort to add a lot of tests or even refactor /v1 code to make those tests possible is wasteful